src/Raven.Server/Documents/Replication/ReplicationDocumentSender.cs

```diff
@@ -786,7 +786,8 @@ private bool ShouldSkip(ReplicationBatchItem item, OutgoingReplicationStatsScope
                         // we let pass all the conflicted revisions, since we keep them with their original change vector which might be `AlreadyMerged` at the destination.
                         if (doc.Flags.Contain(DocumentFlags.Conflicted) ||
                             doc.Flags.Contain(DocumentFlags.FromClusterTransaction) ||
-                            doc.Flags.Contain(DocumentFlags.FromOldDocumentRevision))
+                            doc.Flags.Contain(DocumentFlags.FromOldDocumentRevision) ||
+                            doc.Flags.Contain(DocumentFlags.ForceCreated))
```

Comment:
Why force created revisions aren't replicated? 
Please update the PR description with the root cause to this.